package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eslint/json",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
Looks like it didn't respect [`"bump-minor-pre-major": true`](https://github.com/eslint/json/blob/1976f2c48b1da0cfba2d5ad2553f76182c147621/release-please-config.json#L3) option.

Reply:
Yeah, that's weird. I'll make another PR to reset the version.

Reply:
This could be because of the release-please action we're using.

In this repo, we're using [`google-github-actions/release-please-action@v4`](https://github.com/eslint/json/blob/4fcd8e0a7922b5a9f951061bda2e69cd35ececd2/.github/workflows/release-please.yml#L14). In the rewrite repo, we're using [`GoogleCloudPlatform/release-please-action@v4`](https://github.com/eslint/rewrite/blob/d00cd3eb32fc680c8319414eea3c3023b0011fc8/.github/workflows/release-please.yml#L17). In other repos, e.g. espree, we're using [`google-github-actions/release-please-action@v3`](https://github.com/eslint/espree/blob/ef88e3795fdf8595a71a37020956d05561b1f467/.github/workflows/release-please.yml#L14).

`google-github-actions/release-please-action` is [deprecated](https://github.com/google-github-actions/release-please-action?tab=readme-ov-file#deprecated-release-please-action---please-use-httpsgithubcomgoogleapisrelease-please-action) in favor of `googleapis/release-please-action@v4`.

There seem to be a lot of different versions.

Reply:
Huh, had no idea all that was going on. I guess we should standardize on the new version.